src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -48,6 +48,11 @@ public void accept(ICommandVisitor visitor) {
 				visitor.visit(this);
 			}
 
+			@Override
+			public DriverCommand copy() {
+				return null;
+			}
+
```

Comment:
You shouldn't simply add code not working correctly. Maybe e.g., it is possible to replace anonymous ICompoundCommand() with ComplexCommand implementation.